docs_2.0/manual_EN/3.quick-start/4.nebula-graph-crud.md

```diff
@@ -292,11 +292,11 @@ You can use the statements [GO](TODO) [TODO], [FETCH](TODO) [TODO], and [LOOKUP]
     Got 2 rows (time spent 1935/2420 us)
     ```
 
-* Use `GO`: Search for names and ages of the players whose age is equal to or greater than 35 and that VID "player100" follows. Rename the columns in the result with `Teammate` and `Age`.
+* Use `GO`: Search for the followings of the VID "player100". Filter the followings whose age is equal to or greater than 35. Rename the columns in the result with `Teammate` and `Age`.
```

Comment:
Use followers instead of followings?

Reply:
this is following.  Players whom vid100 follows. 100 -> player